ci.sh

```diff
@@ -12,7 +12,7 @@ elif [ $TEST_SUITE == "integration" ]; then
     eval "$(ssh-agent -s)"
     chmod 400 ./inago-integration-test.pem
     ssh-add ./inago-integration-test.pem 2>/dev/null
-    fab run_int_test
+    fab -A run_int_test
```

Comment:
can we use the long argument version? It's a script, it doesn't cost anything, is more readable and more likely to be future compatible if Fabric decides to change parameters.